### PR TITLE
Implemented Concurrent Maintenance Page

### PR DESCRIPTION
**Pull request by tiwari-nishant**
* Concurrent Maintenance Page Vue2 to Vue3
* Jira Story:https://jsw.ibm.com/browse/PFEBMC-2369

### CODE REVIEW

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js [hunk 1]
```diff
@@ -112,24 +97,25 @@ const ConcurrentMaintenanceStore = {
...
           );
         });
     },
-    async saveReadyToRemoveControlPanel(
-      { commit, state },
-      updatedControlPanel,
-    ) {
-      commit('setReadyToRemoveControlPanel', updatedControlPanel);
+    async saveReadyToRemoveControlPanel(updatedControlPanel) {
+      console.log('rrrrr', this.readyToRemoveControlPanel);
```
Comment (Nikhil-Ashoka) [inline src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js]: Remove console.logs

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js [hunk 2]
```diff
@@ -112,24 +97,25 @@ const ConcurrentMaintenanceStore = {
...
+      this.readyToRemoveControlPanel = updatedControlPanel;
+      console.log('afterrrrrr', this.readyToRemoveControlPanel);
```
Comment (Nikhil-Ashoka) [inline src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js]: Remove console.logs

File: src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue [hunk 3]
```diff
@@ -15,142 +15,153 @@
...
+    concurrentMaintenanceStore.fetchReadyToRemove(),
+    concurrentMaintenanceStore.fetchControlPanel(),
+    concurrentMaintenanceStore.fetchControlPanelDisp(),
+  ]).finally(() => {
+    endLoader();
+  });
+});
+
+function changeReadyToRemoveState(state) {
+  console.log('todoDisp', state);
```
Comment (Nikhil-Ashoka) [inline src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue]: Remove console.log

File: src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue [hunk 4]
```diff
@@ -15,142 +15,153 @@
...
+
+function changeControlPanelState(state) {
+  concurrentMaintenanceStore
+    .saveReadyToRemoveControlPanel(state)
+    .then((message) => successToast(message))
+    .catch(({ message }) => errorToast(message));
+}
+
+function changeControlPanelDispState(state) {
+  console.log('statedisp', state);
```
Comment (Nikhil-Ashoka) [inline src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue]: Remove console.log

File: src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue [hunk 5]
```diff
@@ -1,12 +1,12 @@
 <template>
-  <b-container fluid="xl">
-    <b-row>
-      <b-col md="8" xl="8">
+  <BContainer fluid="xl">
```
Comment (Nikhil-Ashoka) [inline src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue:2]: There are linting error asking to use kebab-case, Not sure why you are getting, Need to check it

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue (hunk 5 above)
Comment (tiwari-nishant) [reply src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue:2]: yeah when I try clicking fix automatically it reverts back to   `<b-container>` . I guess we need to ignore this

Comment (Nikhil-Ashoka) [reply src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue:2]: Yes, I checked when i pulled, But why it is occurring. Need to check?

Comment (Nikhil-Ashoka) [reply src/views/HardwareStatus/ConcurrentMaintenance/ConcurrentMaintenance.vue:2]: No errors now

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js [hunk 6]
```diff
@@ -1,40 +1,27 @@
 import api from '@/store/api';
 import i18n from '@/i18n';
+import { defineStore } from 'pinia';
 
-const ConcurrentMaintenanceStore = {
-  namespaced: true,
-  state: {
-    readyToRemove: false,
+export const ConcurrentMaintenanceStore = defineStore('ConcurrentMaintenance', {
```
Comment (Nikhil-Ashoka) [inline src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js]: Let us keep this defineStore('**ConcurrentMaintenance**' as "**concurrentMaintenance**". C -> lowercase

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js [hunk 7]
```diff
@@ -112,24 +97,23 @@ const ConcurrentMaintenanceStore = {
         })
         .catch((error) => {
           console.log(error);
-          commit('setReadyToRemove', !updatedReadyToRemove);
```
Comment (Nikhil-Ashoka) [inline src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js:115]: This implementation is required, If there is an error message, we need to go back to the current value.
**commit('setReadyToRemove', !updatedReadyToRemove);**

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js (hunk 6 above)
Comment (tiwari-nishant) [reply src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js]: updated

File: src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js (hunk 7 above)
Comment (tiwari-nishant) [reply src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js:115]: updated

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**